data/test/scenarios/test_synced_state.cfg

```diff
@@ -0,0 +1,86 @@
+# wmllint: no translatables
+
+# Check that wesnoth.current.synced_state works as documented, and that a subset of the standard events match the docs
+# in https://wiki.wesnoth.org/EventWML#Multiplayer_safety
+{GENERIC_UNIT_TEST "test_synced_state" (
+    [event]
+        name = preload
+        [lua]
+            code =<<
+                    unit_test.assert_equal(wesnoth.current.synced_state, 'preload', 'preload should have it’s own state')
```

Comment:
its not it's

Reply:
Oops, thanks. Fixed in my local copy, but I'd like to work out what needs fixing in the engine before pushing it again.